applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationAsyncClient.java

```diff
@@ -47,7 +47,7 @@
 public final class ConfigurationAsyncClient extends ServiceClient {
     private static final String SPAN_NAME_TEMPLATE = "Azure.AppConfig/%s";
 
-    private final ServiceLogger logger = new ServiceLogger(ConfigurationAsyncClient.class);
+    private final ClientLogger logger = new ClientLogger(ConfigurationAsyncClient.class);
```

Comment:
Can this be made `static` too? Is a new instance of logger required for every instance of the client? I know this is outside the scope of this PR. I am just trying to understand.

Reply:
Yes, we want a new instance of the logger per new instance of the client. If we had it be static we could conceivably have a single logger instance shared among many different applications all running on the same JVM. More details are here: https://www.slf4j.org/faq.html#declared_static

Reply:
I will make it static.

Reply:
No - I'm saying we do not want it to be static.

Reply:
First I saw many places in our code base that we are using static for logger. But that will cause logger to be shared across many client instances. 
So here I will not use  static. And other places across our code base, we should open a issue to correct that.

Reply:
If there are any places where the logger is static, yes, an issue should be filed to resolve that. Hopefully there are none at this stage.

Reply:
We have static couple places in com.azure.core.http.policy package and many places in com.microsoft.azure.cosmosdb.internal.*  I will file the issue.